magento/magento2
# jshrink package updated to 1.3.0

### Description
Update of https://github.com/tedious/JShrink package to latest version 1.3.0.

The main goal is to bring an ES6 syntax support to Magento.

### Related issues
* https://github.com/tedious/JShrink/issues/66
* https://github.com/tedious/JShrink/issues/69

### Manual testing scenarios
1. Add new or modify existing JS file to utilize ES6 template literals in multiple lines (check the linked issues for code samples)
2. Enable JS minification
3. Do `bin/magento setup:static-content:deploy`
4. The expected result is to successfully finish the process, unexpected is to saw some errors:
   - in normal mode it will looks like this `Sucessful: 2255 files; errors: 2`
   - with `-vvv` flag
```
#0 /magento2/vendor/tedivm/jshrink/src/JShrink/Minifier.php(231): JShrink\Minifier->saveString()
#1 /magento2/vendor/tedivm/jshrink/src/JShrink/Minifier.php(144): JShrink\Minifier->loop()
#2 /magento2/vendor/tedivm/jshrink/src/JShrink/Minifier.php(110): JShrink\Minifier->minifyDirectToOutput('define(function...', Array)
#3 /magento2/vendor/magento/framework/Code/Minifier/Adapter/Js/JShrink.php(27): JShrink\Minifier::minify('define(function...')
#4 /magento2/vendor/magento/framework/View/Asset/PreProcessor/Minify.php(51): Magento\Framework\Code\Minifier\Adapter\Js\JShrink->minify('define(function...')
#5 /magento2/vendor/magento/framework/View/Asset/PreProcessor/Pool.php(74): Magento\Framework\View\Asset\PreProcessor\Minify->process(Object(Magento\Framework\View\Asset\PreProcessor\Chain))
#6 /magento2/vendor/magento/framework/View/Asset/Source.php(152): Magento\Framework\View\Asset\PreProcessor\Pool->process(Object(Magento\Framework\View\Asset\PreProcessor\Chain))
#7 /magento2/vendor/magento/framework/View/Asset/Source.php(105): Magento\Framework\View\Asset\Source->preProcess(Object(Magento\Framework\View\Asset\File))
#8 /magento2/vendor/magento/framework/View/Asset/File.php(150): Magento\Framework\View\Asset\Source->getFile(Object(Magento\Framework\View\Asset\File))
#9 /magento2/vendor/magento/framework/App/View/Asset/Publisher.php(73): Magento\Framework\View\Asset\File->getSourceFile()
#10 /magento2/vendor/magento/framework/App/View/Asset/Publisher.php(61): Magento\Framework\App\View\Asset\Publisher->publishAsset(Object(Magento\Framework\View\Asset\File))
#11 /magento2/vendor/magento/module-deploy/Model/Deploy/LocaleDeploy.php(398): Magento\Framework\App\View\Asset\Publisher->publish(Object(Magento\Framework\View\Asset\File))
#12 /magento2/vendor/magento/module-deploy/Model/Deploy/LocaleDeploy.php(300): Magento\Deploy\Model\Deploy\LocaleDeploy->deployFile('js/input.babel....', 'frontend', 'Snowdog/magento-...', 'en_US', 'Magento_Theme', '/Users/igloczek...')
#13 /magento2/vendor/magento/module-deploy/Model/Deploy/LocaleDeploy.php(229): Magento\Deploy\Model\Deploy\LocaleDeploy->deployAppFiles('frontend', 'Snowdog/magento-...', 'en_US')
#14 [internal function]: Magento\Deploy\Model\Deploy\LocaleDeploy->deploy('frontend', 'Snowdog/magento-...', 'en_US')
#15 /magento2/vendor/magento/framework/App/State.php(171): call_user_func_array(Array, Array)
#16 /magento2/vendor/magento/module-deploy/Model/DeployManager.php(175): Magento\Framework\App\State->emulateAreaCode('frontend', Array, Array)
#17 [internal function]: Magento\Deploy\Model\DeployManager->Magento\Deploy\Model\{closure}(Object(Magento\Deploy\Model\Process))
#18 /magento2/vendor/magento/module-deploy/Model/Process.php(53): call_user_func(Object(Closure), Object(Magento\Deploy\Model\Process))
#19 /magento2/vendor/magento/module-deploy/Model/ProcessManager.php(55): Magento\Deploy\Model\Process->run()
#20 /magento2/vendor/magento/module-deploy/Model/ProcessQueueManager.php(152): Magento\Deploy\Model\ProcessManager->fork(Object(Closure))
#21 /magento2/vendor/magento/module-deploy/Model/ProcessQueueManager.php(130): Magento\Deploy\Model\ProcessQueueManager->fork(Object(Magento\Deploy\Model\ProcessTask))
#22 /magento2/vendor/magento/module-deploy/Model/ProcessQueueManager.php(88): Magento\Deploy\Model\ProcessQueueManager->internalQueueProcess(Array, Array)
#23 /magento2/vendor/magento/module-deploy/Model/DeployManager.php(187): Magento\Deploy\Model\ProcessQueueManager->process()
#24 /magento2/vendor/magento/module-deploy/Model/DeployManager.php(122): Magento\Deploy\Model\DeployManager->runInParallel(Object(Magento\Deploy\Model\DeployStrategyProvider))
#25 /magento2/vendor/magento/module-deploy/Console/Command/DeployStaticContentCommand.php(386): Magento\Deploy\Model\DeployManager->deploy()
#26 /magento2/vendor/symfony/console/Symfony/Component/Console/Command/Command.php(257): Magento\Deploy\Console\Command\DeployStaticContentCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#27 /magento2/vendor/symfony/console/Symfony/Component/Console/Application.php(874): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#28 /magento2/vendor/symfony/console/Symfony/Component/Console/Application.php(195): Symfony\Component\Console\Application->doRunCommand(Object(Magento\Deploy\Console\Command\DeployStaticContentCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#29 /magento2/vendor/magento/framework/Console/Cli.php(96): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#30 /magento2/vendor/symfony/console/Symfony/Component/Console/Application.php(126): Magento\Framework\Console\Cli->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#31 /magento2/bin/magento(23): Symfony\Component\Console\Application->run()
#32 {main}
```
